tjreports/site/models/reports.php

```diff
@@ -1078,15 +1121,15 @@ private function filterReportColumns($queryId, &$selColToshow)
 		}
 
 		$query = $this->_db->getQuery(true);
-		$showhideCols = $paramColToshow = array();
+		$showhideCols = $piiColumns = $paramColToshow = array();
 
 		// Process plugin params
-		$parentId = $this->processSavedReportColumns($queryId, $showhideCols, $paramColToshow, $selColToshow);
+		$parentId = $this->processSavedReportColumns($queryId, $showhideCols, $paramColToshow, $selColToshow, $piiColumns);
```

Comment:
We cannot keep endlessly adding more parameters to the `processSavedReportColumns` method. Please redesign the method more efficiently.

@twsvaishali @manojLondhe